extensions/default/src/DicomWebDataSource/index.js

```diff
@@ -522,10 +522,26 @@ function createDicomWebApi(dicomWebConfig, userAuthenticationService) {
     getConfig() {
       return dicomWebConfigCopy;
     },
+    getStudyInstanceUIDs({ params, query }) {
```

Comment:
we should check this PR if having multiple studyUIDs separated with comma work, this one 

https://docs.ohif.org/configuration/url#studyinstanceuids